core/tee/tee_svc.c

```diff
@@ -211,6 +219,62 @@ TEE_Result tee_svc_sys_get_property(uint32_t prop, tee_uaddr_t buf, size_t blen)
 		return tee_svc_copy_to_user(sess, (void *)buf,
 			&sess->ctx->head->uuid, sizeof(TEE_UUID));
 
+	case UTEE_PROP_TEE_CRYPTOGRAPHY_ECC:
```

Comment:
This switch case is getting a bit too large, it could easily be replaced by a lookup table instead, for example:
```
struct {
     void *data;
     size_t len;
} = props[] = {
     { api_vers, sizeof(api_vers)},
...
};
```
DEV_ID would need to be pre-computed though.